src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -2964,6 +3245,13 @@ namespace ZoneTempPredictorCorrector {
 				SetPointTempSchedIndexHot = SetPointDualHeatCool( SchedTypeIndex ).HeatTempSchedIndex;
 				SetPointTempSchedIndexCold = SetPointDualHeatCool( SchedTypeIndex ).CoolTempSchedIndex;
 				ZoneThermostatSetPointHi( ActualZoneNum ) = GetCurrentScheduleValue( SetPointTempSchedIndexCold );
+
+				//Added Jan 17 (X. Luo)
+				//Adjust operative temperature based on adaptive comfort model
+				if ( ( TempControlledZone( RelativeZoneNum ).AdaptiveComfortTempControl ) ) {
+					AdjustOperativeSetPointsforAdapComfort( RelativeZoneNum, ZoneThermostatSetPointHi( ActualZoneNum ) );
+				}
+
```

Comment:
I prefer to provide an additional warning, when the adaptive comfort thermostat has lower cooling setpoint than one obtained from the original thermostat.